chopper/lib/src/base.dart

```diff
@@ -21,22 +21,25 @@ final allowedInterceptorsType = <Type>[
   DynamicResponseInterceptorFunc,
 ];
 
-/// ChopperClient is the main class of the Chopper API
-/// Used to manager services, encode data, intercept request, response and error.
+/// ChopperClient is the main class of the Chopper API.
+///
+/// It manages registered services, encodes and decodes data, and intercepts
+/// requests and responses.
 class ChopperClient {
-  /// Base url of each request to your api
-  /// hostname of your api for example
+  /// Base URL of each request of the registered services.
+  /// E.g., the hostname of your service.
   final String baseUrl;
 
-  /// Http client used to do request
-  /// from `package:http/http.dart`
+  /// The [http.Client] used to make network calls.
   final http.Client httpClient;
 
-  /// Converter call before request interceptor
-  /// and before interceptor of successful response
+  /// The [Converter] that handles request and response transformation before
+  /// the request and response interceptors are called respectively.
   final Converter converter;
 
-  /// Converter call on error request
+  /// The [ErrorConverter] that handles response transformation before the
+  /// response interceptors are called, but only on error responses
+  /// \(statusCode &lt; 200 || statusCode &gt;= 300\).
```

Comment:
I that okay? :)

Reply:
Well, it renders as `<` and `>`, but the rendering is HTML based, so it should... 😄 But yes, it's a readability issue. I'll fix it in #193 .

Reply:
And  `||` should be `&&` of course. 😅